gatsby/jest.config.js

```diff
@@ -19,4 +19,5 @@ module.exports = {
 	testPathIgnorePatterns: ["/node_modules/", ".cache"],
 	setupFilesAfterEnv: ["./jest.setup.ts"],
 	testURL: "https://cks-gatsby-tests.nice.org.uk/",
+	testEnvironment: "jsdom",
```

Comment:
presume there's nothing wrong with leaving this in here though not needed for 26

Reply:
yeah that's right. I thought I'd leave it in as it's 1 less stumbling block when we do upgrade to 27